README.md

```diff
@@ -26,17 +26,21 @@ Browse the [contributing section](https://github.com/castleproject/Home#its-comm
 
 | Platforms       | Build Status | NuGet Feed |
 |-----------------|-------------:|------------|
-| Windows & Linux | [![Build status](https://ci.appveyor.com/api/projects/status/49a6i0ydiku56r5b/branch/master?svg=true)](https://ci.appveyor.com/project/castleproject/core/branch/master) | [Preview Feed](https://ci.appveyor.com/nuget/core-0mhe40ifodk8)
+| Windows & Linux | [![GitHub Actions](https://github.com/castleproject/Core/workflows/Build%20and%20test/badge.svg?branch=github-actions)](https://github.com/castleproject/Core/actions?query=workflow%3A%22Build%22) | :construction: **TODO** :construction:
```

Comment:
I'd just get rid of the build status because it is integrated into the GitHub UI with the little coloured icons against commits and branches. This just becomes misleading because it is hardcoded to a specific branch.

Reply:
If you want to merge this soon, I'd just get rid of the AppVeyor status badge since we've AppVeyor sends commit status details so this is unneeded. Just keep the preview feed link until we look into GitHub Packages.

Reply:
Done.

I agree with you that merging this sooner rather, then successively iterating on the GitHub Actions workflow(s), is probably better than trying to get everything perfect right from the start.